File: comms/push-notifications/fetch-push-notification-operation/fetch-push-notification-operation.rb
require "twilio"

client = Twilio::Client.new(
  account_id: "<username>",
  auth_token: "<password>"
)

client.push_notifications.fetch_operation(operation_id: "comms_operation_01h9krwprkeee8fzqspvwy6nq8")

File: comms/push-notifications/fetch-push-notification-operation/fetch-push-notification-operation.ts
import { TwilioClient } from "twilio-comms";

async function main() {
    const client = new TwilioClient({
        accountId: "<username>",
        authToken: "<password>",
    });
    await client.pushNotifications.fetchOperation("comms_operation_01h9krwprkeee8fzqspvwy6nq8");
}
main();
